spec/index.html

```diff
@@ -52,13 +53,16 @@
   </script>
 
   <style>
+    body { color-scheme: light; }
+    body.darkmode { color-scheme: dark; }
```

Comment:
```suggestion
```
These should only affect linked SVGs (which this spec does not currently use). Since general support for that will be handled by respec if https://github.com/speced/respec/pull/4925 is merged, removing these lines should happen anyway when that lands, to avoid hard-to-spot regressions (since the upstream fix is not JS-dependent).